docs/pages/api-docs/data-grid/data-grid.md

```diff
@@ -29,6 +29,7 @@ import { DataGrid } from '@material-ui/data-grid';
 | <span class="prop-name">density</span> | <span class="prop-type">Density</span> | standard | Sets the density of the grid. |
 | <span class="prop-name">disableColumnMenu</span> | <span class="prop-type">boolean</span> | false | If `true`, the column menu is disabled. |
 | <span class="prop-name">disableColumnSelector</span> | <span class="prop-type">boolean</span> | false | If `true`, the column selector is disabled. |
+| <span class="prop-name">disableDensitySelector</span> | <span class="prop-type">boolean</span> | false | If `true`, the density selection is disabled. |
```

Comment:
Could we replace "selection" with "selector" in `GridOptions` to make it consistent with the other props?

```suggestion
| <span class="prop-name">disableDensitySelector</span> | <span class="prop-type">boolean</span> | false | If `true`, the density selector is disabled. |
```